compiler/arm-none-eabi-m3/syscfg.yml

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Package: compiler/arm-none-eabi-m3
+
+# syscfg.defs:
+#   HARDFLOAT:
```

Comment:
I don't think there's hardware FPU on Cortex-M3? Given that, this setting should not be here.

Reply:
Hmmm, I thought I had removed all references to `HARDFLOAT`... my bad!

Reply:
Yes, I left it commented out because I thought it could have some side effect... forgot to remove later... thanks!